### ISSUE TITLE
Cannot find symbol device.getType() == AudioDeviceInfo.TYPE_USB_HEADSET

### ISSUE THREAD

**Issue by Jalibk001** [Tintef/react-native-headphone-detection#7]
After installing your package, I see this error after running `npx react-native run-android`

> node_modules\react-native-headphone-detection\android\src\main\java\com\tintef\RNHeadphoneDetection\RNHeadphoneDetectionModule.java:110: error:
>  cannot find symbol
>           device.getType() == AudioDeviceInfo.TYPE_USB_HEADSET
>                                                                      ^
>   symbol:   variable TYPE_USB_HEADSET
>   location: class AudioDeviceInfo`

**Comment by Tintef** [comment]
@Jalibk001 Can you provide a minimum reproducible repo?

**Comment by Jalibk001** [comment]
this just happened after I installed it
Rest there's nothing I can provide

**Comment by Tintef** [comment]
@Jalibk001 You can provide a handful of things.

- node version
- react native version
- npm/yarn version
- environment (mac/windows/linux)
- etc

**Comment by Jalibk001** [comment]
Node version `v12.16.3`

"react-native": "^0.62.2",

npm version `6.14.4`

Development platform is `windows`

**Comment by Tintef** [comment]
I'll try to reproduce with those versions on macos, sadly, I don't own a windows machine.

**Comment by eldareini** [comment]
Same problem, on macOS

System:
    OS: macOS 10.15.5
    CPU: (8) x64 Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz
    Memory: 51.62 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 14.4.0 - /usr/local/bin/node
    Yarn: 1.22.4 - /usr/local/bin/yarn
    npm: 6.14.4 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.9.1 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 13.5, DriverKit 19.0, macOS 10.15, tvOS 13.4, watchOS 6.2
    Android SDK:
      API Levels: 23, 26, 27, 28, 29
      Build Tools: 28.0.0, 28.0.1, 28.0.2, 28.0.3, 29.0.0, 29.0.1, 29.0.2, 29.0.3
      System Images: android-28 | Google Play Intel x86 Atom, android-28 | Google X86_ARM Intel x86 Atom, android-29 | Intel x86 Atom_64, android-29 | Google APIs Intel x86 Atom, android-29 | Google Play Intel x86 Atom
      Android NDK: Not Found
  IDEs:
    Android Studio: 3.6 AI-192.7142.36.36.6241897
    Xcode: 11.5/11E608c - /usr/bin/xcodebuild
  Languages:
    Java: 1.8.0_221 - /usr/bin/javac
    Python: 2.7.16 - /usr/bin/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.11.0 => 16.11.0 
    react-native: 0.62.2 => 0.62.2

**Comment by sirakoff** [comment]
As a temporary fix you could edit `node_modules/react-native-headphone-detection/android/build.gradle` and change compileSdkVersion & buildToolsVersion
```
...
android {
    compileSdkVersion 26
    buildToolsVersion "26.0.1"
    ...
}
```

**Comment by Tintef** [comment]
Closed by #8, merged on master by #9
